src/main/java/project/mybookshop/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package project.mybookshop.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+
+public record UserLoginRequestDto(
```

Comment:
consider using one way of creating DTO: 
1 using plain java object
2 using record

Rewrite code and keep code style consistent